code/__DEFINES/~lobotomy_defines/jobs.dm

```diff
@@ -0,0 +1,89 @@
+#define DEPARTMENT_W_CORP (1<<10)
+#define DEPARTMENT_R_CORP (1<<11)
+#define DEPARTMENT_HANA (1<<12)
+#define DEPARTMENT_ASSOCIATION (1<<13)
+#define DEPARTMENT_SYNDICATE (1<<13)
```

Comment:
It's not the syndicate department, This is for antagonists. 
I am adamant on this; my other maintainers and devs constants bitch about it and are INSISTENT

Reply:
Any suggestions what to name it in that case?